python/available-versions

```diff
@@ -1,181 +1,4 @@
-3.12.0
-3.11.0
 3.10.8
-3.10.7
-3.10.6
```

Comment:
Only the stable and supported versions are left. I also removed version 3.11.0 because the latest azure-cli package has some bugs with this version. We can add this version 3.11.0 again after changing the az command with the extracted msi method.

Reply:
Fine. Only very few versions remain so maybe something to consider for devonfw/ide#941